src/openforms/variables/migrations/0011_migrate_interpolation_format.py

```diff
@@ -0,0 +1,71 @@
+# Generated by Django 3.2.18 on 2023-03-28 12:35
+
+import re
+
+from django.db import migrations
+
+MATCH_SINGLE_BRACKETS = r"{([A-z0-9_]*)}"
+REPL_SINGLE_BRACKETS = r"{{\1}}"
+
+MATCH_DOUBLE_BRACKETS = r"{{([A-z0-9_]*)}}"
+REPL_DOUBLE_BRACKETS = r"{\1}"
+
+
+def apply_regex_nested(data, pattern, repl):
+    if isinstance(data, dict):
+        for key, value in data.items():
+            data[key] = apply_regex_nested(value, pattern, repl)
+    elif isinstance(data, list):
+        data = [apply_regex_nested(value, pattern, repl) for value in data]
+    elif isinstance(data, str):
+        return re.sub(pattern, repl, data)
+    return data
+
+
+def forwards(apps, _):
+    ServiceFetchConfiguration = apps.get_model("variables", "ServiceFetchConfiguration")
+
+    for config in ServiceFetchConfiguration.objects.all():
+        if config.body:
+            config.body = apply_regex_nested(
+                config.body, MATCH_SINGLE_BRACKETS, REPL_SINGLE_BRACKETS
+            )
+        if config.headers:
+            config.headers = apply_regex_nested(
+                config.headers, MATCH_SINGLE_BRACKETS, REPL_SINGLE_BRACKETS
+            )
+        if config.query_params:
+            config.query_params = apply_regex_nested(
+                config.query_params, MATCH_SINGLE_BRACKETS, REPL_SINGLE_BRACKETS
+            )
+
+        config.save()
+
+
+def backwards(apps, _):
+    ServiceFetchConfiguration = apps.get_model("variables", "ServiceFetchConfiguration")
+
```

Comment:
This backwards migration is correct when applied immediately after the forward migration... But after that, people may use template filters in the interpolations. We can either drop these, to get syntactically correct values, but it would change the meaning and results. So I think we should give up on the backwards migration or give a warning that it may break.

Reply:
Hmm that's true, I guess I'll just drop it then